src/lib/mina/actions/offchain-contract.unit-test.ts

```diff
@@ -12,23 +11,22 @@ import assert from 'assert';
 
 const proofsEnabled = true;
 
-const { OffchainState, OffchainStateCommitments } = Experimental;
+const { OffchainState } = Experimental;
 
-const offchainState = OffchainState({
-  accounts: OffchainState.Map(PublicKey, UInt64),
-  totalSupply: OffchainState.Field(UInt64),
-});
+const offchainState = OffchainState(
+  {
+    accounts: OffchainState.Map(PublicKey, UInt64),
+    totalSupply: OffchainState.Field(UInt64),
+  },
+  { logTotalCapacity: 10, maxActionsPerProof: 5 }
```

Comment:
we change the defaults, mainly to check that this works and to exercise the case that settling needs multiple proofs